awscli/examples/mediaconvert/get-job.rst

```diff
@@ -0,0 +1,34 @@
+**To get details for a particular job**
+
+The following example requests the information for the job with ID 1234567890987-1ab2c3, which ended in an error:
+
+Command::
+     aws --endpoint-url=https://abcd1234.mediaconvert.region-name-1.amazonaws.com --region=region-name-1 mediaconvert get-job --id=1234567890987-1ab2c3
+	 
+To get your account-specific endpoint, use describe-endpoints, or send the command without the endpoint. The service will return an error and your endpoint.
+
+Output::
+If your request is successful, the service returns a JSON file with job information, including job settings, any returned errors, and other job data, as follows:
+
+{
```

Comment:
I like the ouputs where you show what it looks like. Is there a reason in some of the examples you don't show the output.

Reply:
Yes, it's because the output includes the request response includes the entire job settings specification, which can be hundreds of lines long. You can see sample job settings here: https://docs.aws.amazon.com/mediaconvert/latest/apireference/sample-json.html

Reply:
Eva, could you possibly show an abbreviated output with the first few lines, then a line with an ellipsis followed by the last few lines?  I also think the lack of output is a bit disconcerting.

@stealthycoin - John, if you could please provide instructions on how to do the rebase from within GitHub, then Eva can finish this submission.

Reply:
I examine the URLs of existing pages to understand what a URL will be when I publish.  They are predicatable.  The link shows up as plain text in the command line since there is no click mechanism there.